projects/cli-files/go-ls/cmd/root.go

```diff
@@ -0,0 +1,60 @@
+package cmd
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"syscall"
+)
+
+func Execute() {
+	defer addFlag()
+	listContent()
+}
+
+func boldText(s string) string {
+	return fmt.Sprintf("\033[1m%s\033[0m", s)
+}
+
+func init() {
```

Comment:
`init` functions are really interesting! Can you describe what they do, and what we need to be careful about when we use them?